src/Microsoft.AspNetCore.App/Microsoft.AspNetCore.App.csproj

```diff
@@ -15,4 +16,8 @@
     <Content Include="lib\$(TargetFramework)\_._" PackagePath="%(Identity)" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.App" Version="[$(RuntimeFrameworkVersion), 2.2.0)" PrivateAsset="None" />
```

Comment:
Which version would we need to target? The bundled version (latest patch) or the pinned version (major.minor.0)?

Reply:
If we add this at all, it should be latest patch. Otherwise, adding this only creates problems and doesn't solve any.